### ISSUE TITLE
Remote node issues - no connection and focus issue

### ISSUE THREAD

**Issue by pingram3541** [bcndev/bytecoin-gui#1]
Tried multiple remote nodes...
```
node2.bytecoin.uk:8081
node3.bytecoin.uk:8081
bcn-node01.bytecoin.cloud:8081
bcn-node02.bytecoin.cloud:8081
bcn-node03.bytecoin.cloud:8081
node.bytecoin.ninja:8081
```

However none connect and cause more problems on top of not connecting.

For example, the first two nodes simply loop and report the following in the console:

```
Bytecoin GUI is in detached state.
You can open a wallet file, create a new one, or connect to remote walletd daemon.
--> {"id":"0","jsonrpc":"2.0","method":"get_addresses","params":{}}

```

Each time it does this, every 2-3 seconds, the wallet app steals focus so it is extremely difficult to use any other app running on my MAC unless I close the wallet completely first.  Additionally, it steals focus from it's own dialogs making it difficult to access the menu and choose to open a wallet as the dialog interfere's with browsing folders to open a wallet file having to click inside the window again.

Even worse is when I use the other bytecoin.cloud nodes, (bcn-nodeX...) it results in an error dialog saying "Method Not Found" and the following is shown in the console:

```
Connecting to walletd...
--> {"id":"0","jsonrpc":"2.0","method":"get_addresses","params":{}}
<-- {"error":{"code":-32601,"message":"Method not found"},"id":"0","jsonrpc":"2.0"}
```

Just like with the bytecoin.uk nodes, this occurs every 2-3 seconds but makes any open dialog of the wallet disappear (it's own dialog windows!), making it impossible to edit the node or open a wallet file.  I had to close the wallet app, manually edit and remove the node from the cfg file to be able to access the GUI again.

If I choose to open a wallet file instead first, it ignores the remote node settings and connects using the local daemon and downloads the entire blockchain which is what I am trying to avoid.  40gb is pushing the boundaries for a simple wallet IMO and should not be necessary for me to own 5+ years of ledger I will never use.

Once the wallet is open, if I choose remote node again, the wallet is immediately closed and the connection loop begins.  When looking at the configs, the remote nodes are stored as "recentWallets"???  Not sure why a node would be called a wallet but maybe nodes are handled differently in the beta wallet and you just haven't told us how to use it properly???

Also there are no instructions, at least that I could find easily except for old outdated stuff, where can we find documentation that applies to the new wallet?

Lastly, one cannot remove a node through the GUI.  This is a design flaw I think.  Once it's been saved, there is no deleting it via the GUI it and you must manually edit the cfg file.

**Comment by bcndev** [comment]
@pingram3541 
1. Focus stealing issue will be fixed in the next beta release
2. As to connecting issue, notice that Bytecoin beta software is built upon new API and does not support the older versions of API. So, according to you report, it looks like you just connected to old Bytecoin nodes.

**Comment by pingram3541** [comment]
@bcndev 

1. Thx =)
2. So is there any documented nodes using the new API that we can use now?

What about documentation for the folks running the old nodes on how to get them to support the new beta wallet version.  Can or would it even be necessary for them to support both at the same time or is it put together in a way that once the node is adjusted to use the new API format/method that the old format/method will automatically work?

**Comment by bcndev** [comment]
@pingram3541 Up-to-date documentation is supposed to be located in our [Wiki](https://github.com/bcndev/bytecoin/wiki). It's now not finished but fairly enough to get started integration with new API. We'd recommend consider transition solely to the New Bytecoin Services right away.

**Comment by pingram3541** [comment]
Thx, I was able to connect to the API and test it.  I think I will change the port =)

Is remote node fully functional and if so, which do you suggest using?

**Comment by bcndev** [comment]
Focus stealing is fixed in fc372ccfcc9c4505d3cd7857c30632a05bda54a8.
PS: No clue about nodes you listed.

**Comment by pingram3541** [comment]
I was hoping there was a new remote node we can start using that supports the new API, similar to the previous wallet for those of us that prefer a "light" wallet, i.e. ssd, limited space
